src/status_im/protocol/handlers.cljs

```diff
@@ -320,9 +325,9 @@
                                  :type       type
                                  :ttl        (+ (datetime/now-ms) ttl-s)}
               route-event (case type
-                            :message               [:received-protocol-message! message]
-                            :group-message         [:received-protocol-message! message]
-                            :public-group-message  [:received-protocol-message! message]
+                            :message               [:received-message (transform-protocol-message message)]
```

Comment:
can we just use `(:message :group-message :public-group-message)` here?

Reply:
In the case statement, strict equality check `=` is only supported, so doing what you say would require to rewrite it using `cond`/`condp`

Reply:
are you sure?
```
(case 'y
      (x y z) "x, y, or z"
      "default")
;;=> "x, y, or z"
```
https://clojuredocs.org/clojure.core/case#example-542692ccc026201cdc326c89

Reply:
My Clojure knowledge is super rusty, everyday, I learn something new :) 
I will fix it off course.